unity-game/Assets/Scripts/ElevatorScripts/AutoElevator.cs

```diff
@@ -0,0 +1,220 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class AutoElevator : MonoBehaviour
+{
+    public bool notMoving;
+    public bool reachedDestination;
+    public bool onStartPoint;
+    public bool onEndPoint;
+
+    public GameObject objectToMove;
+    public GameObject theRightWall;
```

Comment:
for some reason, it glitched out when I didn't have the "the"s. I forget why it didn't like that, but I added in the "the"s and it never got mad again.